core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -346,7 +346,7 @@ private static RelDataType deriveTypePad(SqlOperatorBinding binding, RelDataType
   @LibraryOperator(libraries = {BIG_QUERY, ORACLE, POSTGRESQL, SPARK, HIVE})
   public static final SqlFunction LTRIM =
       SqlBasicFunction.create(SqlKind.LTRIM,
-          ReturnTypes.ARG0.andThen(SqlTypeTransforms.TO_NULLABLE)
+          ReturnTypes.ARG0.andThen(SqlTypeTransforms.FORCE_NULLABLE)
```

Comment:
Is it only possible to change this in the Oracle mode here?

Reply:
Yes, this makes the functions return null even if no argements is null.